packages/fast-components-react-base/src/breadcrumb-item/breadcrumb-item.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { get } from "lodash-es";
+import Hypertext, { HypertextClassNameContract } from "../hypertext";
+import { BreadcrumbItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    BreadcrumbItemHandledProps,
+    BreadcrumbItemProps,
+    BreadcrumbItemUnhandledProps,
+} from "./breadcrumb-item.props";
+
+class BreadcrumbItem extends Foundation<
+    BreadcrumbItemHandledProps,
+    BreadcrumbItemUnhandledProps,
+    {}
+> {
+    public static displayName: string = "BreadcrumbItem";
+
+    public static defaultProps: Partial<BreadcrumbItemHandledProps> = {
+        current: false,
+    };
+
+    protected handledProps: HandledProps<BreadcrumbItemHandledProps> = {
+        managedClasses: void 0,
+        children: void 0,
+        current: void 0,
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLLIElement> {
+        return (
+            <li {...this.unhandledProps()} className={this.generateClassNames()}>
+                <Hypertext

```

Comment:
What if someone needed these to be buttons? I don't know that we should hard require hypertext here.

Reply:
semantically though, these would always need to be anchors I think

Reply:
I feel like we could find an outlier that was valid, but either way - we likely should follow predefined patterns and not hard code our own components here.

I think this should likely just be children.

Reply:
According to w3 these need to be anchor tags and aria-current needs to be set to page on the last item

Reply:
In MSFT we can put hypertext in as our examples/patterns.